### ISSUE TITLE
Xamaridea Mac

### ISSUE THREAD

**Issue by benoitjadinon** [EgorBo/Xamaridea#4]
I'll try to port your tool to Mac, because the Xamarin Studio axml editor is really bad over here too ;-)
thanks for thinking about separating projects between the sync / plugin.

**Comment by EgorBo** [comment]
@benoitjadinon looking forward to trying that :smile:

**Comment by benoitjadinon** [comment]
Train ride home R&D FTW ;) https://github.com/benoitjadinon/Xamaridea/commit/7f52d43cbac16f779c76509726fa1a3d5a3b1a9b
it works from the command line on mac now.
but I'd like to add more features, like : 
- benoitjadinon#1 automatic fill of the andro sdk in local.properties 
- benoitjadinon#2 more configurations (gradle options like versions, ...) through more cmd arguments
- benoitjadinon#3 automatic delete of the temp project after AS is closed
- benoitjadinon#4 have the option to use a non-embedded template (not as a zip but as a folder somewhere) so that maybe you'd have a custom template with custom settings for a specific project
  - benoitjadinon#5 also maybe have the template not-zipped, so that changes we do in its files are seen through git
    ...
- benoitjadinon#6 more asynchronicity (not so useful for the console app, but for the future XS plugin)
- benoitjadinon#7 try at a Xamarin Studio plugin
- benoitjadinon#8 cleanup the tabs indentation ;-)

**Comment by benoitjadinon** [comment]
https://github.com/benoitjadinon/Xamaridea
commandline now works pretty well on mac, gotta switch to windows to test

**Comment by MKuckert** [comment]
There is currently no release for Xamarin Studio (or Visual Studio for Mac), right?

**Comment by benoitjadinon** [comment]
nope, currently trying to reboot the project as netstandard command-line tool
